index.js

```diff
@@ -80,15 +85,15 @@ exports.remapCoverage = function(coverage, originalSources) {
   originalSources = originalSources || _originalSources;
   var collector = remapIstanbul(coverage, {
     readFile: function(name) {
-      return originalSources[name].source + 
-          (originalSources[name] && originalSources[name].sourceMap ? '\n//# sourceMappingURL=' + name.split('/').pop() + '.map' : '');
+      return originalSources[name].source +
+          (originalSources[name] && originalSources[name].sourceMap ? '\n//# sourceMappingURL=' + name.split(path.sep).pop() + '.map' : '');
```

Comment:
This worries me a little bit because we're supposed to have URLs in the registry, so we should already be using `/` in the name not `\\` at all. Perhaps removing the `path.normalize` above will remove the need for this?